Photo App Screen/HeaderView.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+
+class HeaderView: UICollectionReusableView {
+
+    static let headerViewID = "HeaderView"
+
+    let label = UILabel()
+    let subView = UIView()
+    let button: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle("См. все", for: .normal)
+        button.titleLabel?.textColor = .systemBlue
+        button.titleLabel?.font = UIFont.preferredFont(forTextStyle: .title3)
+        return button
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+}
+
+extension HeaderView {
+
+    func configure() {
+
+        backgroundColor = .systemBackground
+
+        addSubview(label)
+        addSubview(subView)
+        addSubview(button)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.adjustsFontForContentSizeCategory = true
+        subView.translatesAutoresizingMaskIntoConstraints = false
+        button.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
можно положить в массив и через замыкания форича написать - лаконично будет
```suggestion
        label.adjustsFontForContentSizeCategory = true

        [label, subView, button].forEach {
            addSubview($0)
            $0.translatesAutoresizingMaskIntoConstraints = false
        }
```